src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.util.Arrays;
+
+@Entity
```

Comment:
add name for new table

Reply:
yes, i know that i must to do, but i just forgor